frontend/types/subscriptionModal.ts

```diff
@@ -0,0 +1,74 @@
+import { ChainIDs } from './network'
+
+/** translates the names of the members of our objects to/from the names used by the API  // TODO: write here the identifiers actually used by the API */
+export enum SubscriptionJSONfields {
```

Comment:
> This mapping makes the whole process quit complex ... I think it would be more straight forward to use the backend API structs directly.

I agree (I did not know how to do and I did not like my solutions).
Changed.
Now the component use the JSON of _types/api/notifications.ts_ (as it is not available to us yet, I copied its prototypes into _types/subscriptionModal.ts_)